main.go

```diff
@@ -11,7 +14,25 @@ const (
 	gitFilterRepoURL    = "https://raw.githubusercontent.com/newren/git-filter-repo/" + gitFilterRepoCommit + "/git-filter-repo"
 )
 
-type Git struct{}
+type Git struct {
+	// +private
+	KnownHosts []string
+}
+
+func New(
+	// +optional
+	knownHost []string,
```

Comment:
I'm trying to figure out if string slices can be passed in as multiple options or as a single one separated by commas.